packages/m/mirrorx.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "mirrorx",
+  "filename": "umd/mirror.js",
```

```suggestion
  "filename": "mirror.js",
```
The file in `dist` is just `mirror.js`